spec/unit/provider/package/rpm_spec.rb

```diff
@@ -7,12 +7,13 @@
 
   let (:packages) do
     <<-RPM_OUTPUT
-    cracklib-dicts 0 2.8.9 3.3 x86_64
+    'cracklib-dicts 0 2.8.9 3.3 x86_64
```

Comment:
the quote is being added in the provider here:
https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/package/rpm.rb#L134
which when passed to rpm via argv (where each argument is it's own string index) it gets retained in the output:

rpm -q --qf "'%{NAME} %|EPOCH?{%{EPOCH}}:{0}| %{VERSION} %{RELEASE} %{ARCH}\\n'" -p ~/c/vagrant-pe4-simp/rpms/simp-adapter-pe-0.0.1-0.noarch.rpm 
'simp-adapter-pe 0 0.0.1 0 noarch
'